### PR TITLE
[Misc] Update GitHub template

### PR DESCRIPTION
**Pull request by hydai**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+Fork the WasmEdge repository on GitHub to your personal account.
+
+
+```
+$ git clone git@github.com:WasmEdge/WasmEdge.git
+$ cd WasmEdge
+```
+
+Notes: Note the WasmEdge team builds lots of extensions of Server-side WebAssembly, see [TensorFlow](https://github.com/second-state/SSVM-tensorflow), [Storage](https://github.com/second-state/SSVM-storage), [Command interface](https://github.com/second-state/ssvm_process_interface), [Ethereum](https://github.com/second-state/SSVM-evmc), [Substrate](https://github.com/second-state/substrate-ssvm-node). If you want to contribue the extensions, please go to those repositories.
```
Comment (q82419) [inline .github/CONTRIBUTING.md]: TensorFlow: https://github.com/second-state/WasmEdge-tensorflow
Storage: https://github.com/second-state/WasmEdge-storage
Command interface: https://github.com/second-state/wasmedge_process_interface
Ethereum: https://github.com/second-state/WasmEdge-evmc

**Review by q82419 — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (0yi0) [reply .github/CONTRIBUTING.md]: This should still be fixed but in a different file `docs/CONTRIBUTING.md`.

**Review by 0yi0 — CHANGES_REQUESTED** [review]
Links and paths to `doc/` should also be updated.
- `README*.md`
- `utils/witx-doc/generate.sh`
- `Changelog.md` *# I'm not sure about this one.*

**Review by ibmibmibm — DISMISSED** [review]

**Comment by hydai** [comment]
Just fixed them all. CC @0yi0

**Review by 0yi0 — DISMISSED** [review]

**Review by 0yi0 — COMMENTED** [review]
As #327 is being merged, `.github/workflows/linter.yml` should also be updated.

**Comment by hydai** [comment]
> As #327 is being merged, `.github/workflows/linter.yml` should also be updated.

Updated.

**Review by q82419 — APPROVED** [review]

**Review by q82419 — APPROVED** [review]

**Pull request merged**